### PR TITLE
Update number_format.d.ts

### PR DESCRIPTION
**Pull request by IgnorantCoder**
Relates to #220

### CODE REVIEW

File: typings/number_format.d.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ declare module "react-number-format" {
     type?: "text" | "tel";
     isAllowed?: (values: NumberFormatValues) => boolean;
     renderText?: (formattedValue: string) => React.ReactNode;
-    getInputRef?: (el: HTMLInputElement) => void;
+    getInputRef?: ((el: HTMLInputElement) => void) | React.Ref<any> | string;
```
Comment (s-yadav) [inline typings/number_format.d.ts]: Passing getInputRef as string will break. Why do we want to keep that as option ?

Comment (IgnorantCoder) [reply typings/number_format.d.ts]: Oh sorry.
I think supporting string is not good, too.
I want to support React.Ref<any> only.

**Comment by s-yadav** [comment]
Looks good. Can you squash your commits ?

**Comment by s-yadav** [comment]
Merged now. Thanks.

**Pull request merged**